aten/src/ATen/native/cuda/UpSampleTrilinear3d.cu

```diff
@@ -234,7 +254,6 @@ static void upsample_trilinear3d_out_cuda_template(
                   output_depth,
                   output_height,
                   output_width});
-  output.zero_();
```

Comment:
was zero unneeded here because all output elements were written to and not read from?

Reply:
Yes, that's right. See also https://github.com/pytorch/pytorch/pull/21879/files#diff-2fdaa774b92aedf3bde904acccf6d10e8763d7987ccc8ad771925b1c144bb283L179